workflows/taxprofiler.nf

```diff
@@ -150,6 +150,9 @@ workflow TAXPROFILER {
     // Validate and decompress databases
     ch_dbs_for_untar = databases
         .branch { db_meta, db_path ->
+            if ( !db_meta.db_type ) {
+                db_meta = db_meta + [ db_type: "short;long" ]
```

Comment:
@jfy133 Yes, we add `db_type` and set it as `short;long` if the `db_type` column is missing in `database.csv`. I have tested the pipeline with both [version 1](https://github.com/nf-core/test-datasets/blob/taxprofiler/database_full_v1.1.csv) and [version 2 ](https://github.com/nf-core/test-datasets/blob/taxprofiler/database_full_v1.2.csv)test data